definitions/npm/history_v4.x.x/test_history_v4.x.x.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
Put the test file inside the flow version folder. Tests may vary based on flow version.

Reply:
I tried moving the test file into the specific flow version folder and I got an error on one of the regression tests. It seems like there can't not be a common test file for the all versions

Reply:
Hmm it seems to be failing on the regression test `regression-1385_v1.x.x` just intermittently, not related to any changes I made. I guess I got lucky the first time

Reply:
Ah it looks like there was a change to the error messages in flow 0.66.0 which broke the regression test

Reply:
Once the issue with flow 0.66.0 resolves itself, can you try moving the test file again?

Reply:
yeah i'll do that. I'll save it until master is green again for triggering travis